src/fragments/lib-v1/analytics/native_common/getting-started/common.mdx

```diff
@@ -104,6 +124,5 @@ Next Steps:
 Congratulations! Now that you have Analytics' backend provisioned and Analytics library installed.  Check out the following links to see Amplify Analytics use cases:
 
 * [Record Events](/lib-v1/analytics/record)
-* [Track Sessions](/lib-v1/analytics/autotrack)
 * [Identify User](/lib-v1/analytics/identifyuser)
-* [Escape Hatch](/lib-v1/analytics/escapehatch)
+* <InlineFilter filters={["android", "ios", "flutter"]}>[Track Sessions](/lib-v1/analytics/autotrack)</InlineFilter><InlineFilter filters={["js"]}>Track Sessions - Coming soon!</InlineFilter>
```

Comment:
Currently links can't be inlined with other react components.  Doing so will make the link rendered as plain text
![Screenshot 2023-09-25 at 12 10 49 PM](https://github.com/aws-amplify/docs/assets/7351516/d3f32e43-5260-45d6-91c7-b91b4c2af60d)

instead need to do something like this
```suggestion
<InlineFilter filters={["android", "ios", "flutter"]}>

* [Record Events](/lib-v1/analytics/record)
* [Identify User](/lib-v1/analytics/identifyuser)
* [Track Sessions](/lib-v1/analytics/autotrack)

</InlineFilter>
<InlineFilter filters={["js"]}>

* [Record Events](/lib-v1/analytics/record)
* [Identify User](/lib-v1/analytics/identifyuser)
* Track Sessions - Coming soon!

</InlineFilter>
```

Reply:
Hum it seems to be rendering fine in the preview, I'll make the change though

Reply:
the js-v6 page doesn't have a link in it, its the ios, android, and flutter versions of the page that aren't rendering correctly
https://amplify-js-v6.dmhjrabi3pkql.amplifyapp.com/lib-v1/analytics/getting-started/q/platform/ios/#view-analytics-console